tests/llmcompressor/transformers/finetune/test_finetune_no_recipe_custom_dataset.py

```diff
@@ -122,7 +123,14 @@ class TestOneshotCustomDatasetSmall(TestFinetuneNoRecipeCustomDataset):
     def setUp(self):
         import torch
 
-        self.device = "cuda:0" if torch.cuda.is_available() else "cpu"
+        self.monkeypatch = pytest.MonkeyPatch()
+
+        if torch.cuda.is_available():
+            self.device = "cuda:0"
+            self.monkeypatch.setenv("CUDA_VISIBLE_DEVICES", "0")
+        else:
+            self.device = "cuda:0"
```

Comment:
if cuda is availble set to the first recognized - ok
else set to the first recognized? should this be cpu?

Reply:
oops typo. good catch

Reply:
Yes, this should be cpu
```suggestion
            self.device = "cpu"
```